### PR TITLE
fix-artifacts-finder

### PR DESCRIPTION
**Pull request by davidryan59**
Contains some minor fixes for Artifacts Finder:
- Fix potential bug where the prospecting loop time was controlled, but the finding loop time was not controlled
- Make it easier for user to identify and amend the two main parameters
- Minor tidying and tweaks

### CODE REVIEW

File: content/artifacts/artifacts-finder/plugin.js [hunk 1]
```diff
@@ -168,8 +177,8 @@ class ArtifactsFinder {
       this.logs.appendChild(createDom("div", "Start Finding"));
       this.prospectArtifacts();
       setTimeout(this.findArtifacts.bind(this), 0);
-      this.prospectTimerId = setInterval(this.prospectArtifacts.bind(this), AUTO_INTERVAL);
-      this.findTimerId = setInterval(this.findArtifacts.bind(this), 10000);
+      this.prospectTimerId = setInterval(this.prospectArtifacts.bind(this), LOOP_INTERVAL_MS);
+      this.findTimerId = setInterval(this.findArtifacts.bind(this), LOOP_INTERVAL_MS);
```
Comment (snowtigersoft) [inline content/artifacts/artifacts-finder/plugin.js:181]: Do not change this line, find should start asap

Comment (davidryan59) [reply content/artifacts/artifacts-finder/plugin.js:181]: OK thanks for letting me know. I'll revert it.

Comment (davidryan59) [reply content/artifacts/artifacts-finder/plugin.js]: Added commit to revert the second line above, marking as resolved

**Comment by davidryan59** [comment]
@snowtigersoft can you help me understand why the Prospecting is on a slow loop (5 or 2 minutes) but the Finding is on a fast loop (10 seconds)?

Without the edit above, this PR is no longer fixing any bug, it becomes cosmetic in nature

@snowtigersoft do you think this PR adds anything in terms of making it easier for user to control parameters? If not, I'm happy to close the PR without merge. Thank you for helping me understand the plugin!

**Comment by snowtigersoft** [comment]
@davidryan59 because you can prospecting at any time after the planet can prospect, BUT MUST find in 256 blocks after prospected, after 256 blocks, you lose this artifact forever.

I think is not very necessary to change the parameters' name, because 99% users do not need to change it.

**Comment by Stx69** [comment]
@davidryan59 In my best experience I use main loop 3mins & finding loop any 10s... I'm not sure but additional parameter what a like is how many max task to queue.,.

**Comment by davidryan59** [comment]
OK. I'm closing this PR since it sounds like folks don't need either the "bug fix" or the cosmetic changes. Thanks for the comments anyway!

**Pull request closed without merging**